homeassistant/components/garmin_connect/sensor.py

```diff
@@ -0,0 +1,255 @@
+    "lastSevenDaysAvgRestingHeartRate": [
+        "Last 7 Days Avg Heart Rate",
+        "bpm",
+        "mdi:heart-pulse",
+    ],
+    "averageStressLevel": ["Avg Stress Level", "", "mdi:flash-alert"],
+    "maxStressLevel": ["Max Stress Level", "", "mdi:flash-alert"],
+    "stressQualifier": ["Stress Qualifier", "", "mdi:flash-alert"],
+    "stressDuration": ["Stress Duration", "minutes", "mdi:flash-alert"],
+    "restStressDuration": ["Rest Stress Duration", "minutes", "mdi:flash-alert"],
+    "activityStressDuration": [
+        "Activity Stress Duration",
+        "minutes",
+        "mdi:flash-alert",
+    ],
+    "uncategorizedStressDuration": [
+        "Uncat. Stress Duration",
+        "minutes",
+        "mdi:flash-alert",
+    ],
+    "totalStressDuration": ["Total Stress Duration", "minutes", "mdi:flash-alert"],
+    "lowStressDuration": ["Low Stress Duration", "minutes", "mdi:flash-alert"],
+    "mediumStressDuration": ["Medium Stress Duration", "minutes", "mdi:flash-alert"],
+    "highStressDuration": ["High Stress Duration", "minutes", "mdi:flash-alert"],
+    "stressPercentage": ["Stress Percentage", "%", "mdi:flash-alert"],
+    "restStressPercentage": ["Rest Stress Percentage", "%", "mdi:flash-alert"],
+    "activityStressPercentage": ["Activity Stress Percentage", "%", "mdi:flash-alert"],
+    "uncategorizedStressPercentage": [
+        "Uncat. Stress Percentage",
+        "%",
+        "mdi:flash-alert",
+    ],
+    "lowStressPercentage": ["Low Stress Percentage", "%", "mdi:flash-alert"],
+    "mediumStressPercentage": ["Medium Stress Percentage", "%", "mdi:flash-alert"],
+    "highStressPercentage": ["High Stress Percentage", "%", "mdi:flash-alert"],
+    "moderateIntensityMinutes": ["Moderate Intensity", "minutes", "mdi:flash-alert"],
+    "vigorousIntensityMinutes": ["Vigorous Intensity", "minutes", "mdi:run-fast"],
+    "intensityMinutesGoal": ["Intensity Goal", "minutes", "mdi:run-fast"],
+    "bodyBatteryChargedValue": [
+        "Body Battery Charged",
+        "%",
+        "mdi:battery-charging-100",
+    ],
+    "bodyBatteryDrainedValue": [
+        "Body Battery Drained",
+        "%",
+        "mdi:battery-alert-variant-outline",
+    ],
+    "bodyBatteryHighestValue": ["Body Battery Highest", "%", "mdi:battery-heart"],
+    "bodyBatteryLowestValue": ["Body Battery Lowest", "%", "mdi:battery-heart-outline"],
+    "bodyBatteryMostRecentValue": [
+        "Body Battery Most Recent",
+        "%",
+        "mdi:battery-positive",
+    ],
+    "averageSpo2": ["Average SPO2", "%", "mdi:diabetes"],
+    "lowestSpo2": ["Lowest SPO2", "%", "mdi:diabetes"],
+    "latestSpo2": ["Latest SPO2", "%", "mdi:diabetes"],
+    "latestSpo2ReadingTimeLocal": ["Latest SPO2 Time", "", "mdi:diabetes"],
+    "averageMonitoringEnvironmentAltitude": [
+        "Average Altitude",
+        "%",
+        "mdi:image-filter-hdr",
+    ],
+    "durationInMilliseconds": ["Duration", "ms", "mdi:progress-clock"],
```

Comment:
What duration is this?

Reply:
Unknown yet, I removed it as devicetype for now.